lib/doorkeeper/oauth/refresh_token_request.rb

```diff
@@ -32,7 +33,7 @@ def load_client(credentials)
       def before_successful_response
         refresh_token.transaction do
           refresh_token.lock!
-          raise Errors::InvalidTokenReuse if refresh_token.revoked?
+          raise Errors::MissingRequiredParameter, :refresh_token if refresh_token.revoked?
```

Comment:
Actually I don't sure it's a valid behavior to return `MissingRequiredParameter` in case refresh token exist, but revoked. Does RFC say something like that? I think `MissingRequiredParameter` must indicate only errors when parameter is **really** missed

Reply:
I fixed it here: https://github.com/doorkeeper-gem/doorkeeper/pull/1277/commits/8fbf46ca61ed7e8f2fa237129e4be31976875ee1